src/app/seamlyme/dialogs/dialogmdatabase.cpp

```diff
@@ -140,50 +143,31 @@ DialogMDataBase::~DialogMDataBase()
 }
 
 //---------------------------------------------------------------------------------------------------------------------
-QStringList DialogMDataBase::GetNewNames() const
+QStringList DialogMDataBase::GetNewMeasurements() const
```

Comment:
Thanks for renaming GetNewNames() to GetNewMeasurements().

Reply:
> Thanks for renaming GetNewNames() to GetNewMeasurements().

Would not getNewMeasurementNames() make more sense?  Your getting a list of names not measurements... 

QStringList newNames;
....
return newNames;

You're just adding more confusion by having GetNewMeasurements() return newNames. 

By using getNewMeasurementNames() returning newNames the two jive.